### PR TITLE
hammerhead version updated (closes #349)

### LINKED ISSUE #349: Update `testcafe-hammerhead` to v6.0.0
There is a breaking change in the https://github.com/DevExpress/testcafe-hammerhead/issues/410 PR. We should modify our code according to this.

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\CC @AlexanderMoskovkin 
PR

### CODE REVIEW

File: src/client/runner/step-iterator.js [hunk 1]
```diff
@@ -319,10 +319,13 @@ StepIterator.prototype._checkIFrame = function (element, callback) {
 
     var iFrame = domUtils.getIframeByElement(element);
 
-    this.pingIframe(iFrame, function () {
-        callback(iFrame);
-    });
-
+    this.pingIframe(iFrame)
```
Comment (AlexanderMoskovkin) [inline src/client/runner/step-iterator.js]:
```js
this
    .pingIframe(iFrame)
    .then(() => callback(iFrame))
    .catch(() => callback(iFrame));
```

Comment (AlexanderMoskovkin) [inline src/client/runner/step-iterator.js]: /r-

**Comment by georgiy-abbasov** [comment]
FPR
\cc @miherlosev, take a look, please

File: src/client/runner/runner-base.js [hunk 2]
```diff
@@ -373,18 +371,17 @@ RunnerBase.prototype._runInIFrame = function (iframe, stepName, step, stepNum) {
...
     }
 
-    pingIframe(iframe, function (err) {
-        if (err) {
+    pingIframe(iframe)
+        .then(() => {
+            runner.iframeExistenceWatcherInterval = window.setInterval(iframeExistenceWatcher, IFRAME_EXISTENCE_WATCHING_INTERVAL);
+            messageSandbox.sendServiceMsg(msg, iframe.contentWindow);
+        })
+        .catch(() => {
```
Comment (miherlosev) [inline src/client/runner/runner-base.js]: It is wrong.
Need to rewrite without `catch`.
see commit https://github.com/miherlosev/testcafe-hammerhead/commit/28757401aaea8709544bd150eb50e83887758026

File: src/client/runner/step-iterator.js (hunk 1 above)
Comment (miherlosev) [inline src/client/runner/step-iterator.js]: Without `catch`.
see https://github.com/miherlosev/testcafe-hammerhead/commit/28757401aaea8709544bd150eb50e83887758026

Comment (AlexanderMoskovkin) [inline src/client/runner/step-iterator.js]: > Without catch.

I can't understand what do you mean. If `pingIframe` is rejected we should call the `callback` too

**Comment by miherlosev** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm
I'll merge this when tests are done

**Comment by georgiy-abbasov** [comment]
@AlexanderMoskovkin, thanks

**Pull request merged**